packages/salesforcedx-vscode-core/test/vscode-integration/commands/isvdebugger/bootstrapCmd.test.ts

```diff
@@ -19,7 +19,8 @@ import { getRootWorkspacePath } from '../../../../src/util';
 describe('ISV Debugging Project Bootstrap Command', () => {
   const LOGIN_URL = 'a.b.c';
   const SESSION_ID = '0x123';
-  const PROJECT_NAME = 'sfdx-simple';
```

Comment:
Since we are now wiping out the project prior to executing the ISV Debugger, we will copy the contents of 'sfdx-simple' into a new clone with the stub project setup call.